vignettes/extract_convert.Rmd

```diff
@@ -0,0 +1,275 @@
+---
+title: "Parameter extraction and conversion in {epiparameter}"
+output: 
+  bookdown::html_vignette2:
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: references.json
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Parameter extraction and conversion in {epiparameter}}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The reporting of parameters of distributions in the epidemiological literature 
+is varied. In some instances the parameters themselves will be given, in others,
+summary statistics of the distribution are provided, such as mean, standard 
+deviation, variance, among others. 
+
+When working with epidemiological parameters it can be useful to have the values
+of the parameters for common parameterisations of the distribution. For example,
+shape and scale parameters for the gamma and Weibull distribution, and the meanlog
+and sdlog parameters for the lognormal distribution. 
+
+In {epiparameter} we provide a range of conversion and extraction functions in
+order to obtain distribution parameters from summary statistics and vice versa.
+
+Here we differentiate between conversion and extraction as what can be analytically
+converted, as conversions, and those that require optimisation as extraction. 
+This vignette aims to describe the functionality of these conversions and 
+extractions provided by the {epiparameter} package.
+
+::: {.alert .alert-info}
+### Conversion versus extraction
+
+Use conversion when possible over extraction to avoid possible limitations associated with
+numerical optimisation used in the extraction function `extract_param()`.
+:::
+
+```{r setup}
+library(epiparameter)
+```
+
+## Conversions
+
+There are two conversion functions in {epiparameter}: `convert_params_to_summary_stats()` and `convert_summary_stats_to_params()`. `convert_params_to_summary_stats()` converts one set of statistical distribution parameters to common summary statistics, and `convert_summary_stats_to_params()` converts summary statistics to a set of parameters.
+
+### Conversion functions
+
+The conversion functions are generic functions. That means that different types of data can be supplied to the first argument and the correct operation (method) will be executed. Currently {epiparameter} provides conversion methods for `character` and `<epidist>`. If this is unclear, see the examples below for how the conversion functions can work when you supply either a `character` string or an `<epidist>`.
```

Comment:
Once again, I think this is too technical for the general audience. All they need to use this function is that they can supply the string of the distribution name or an epidist.

```suggestion
The conversion functions can take two types of inputs as the first argument: a `character` string of the distribution or an `<epidist>` object.
```